index.d.ts

```diff
@@ -1,7 +1,12 @@
+type MethodTypes = 'widget-opened' | 'widget-closed' | 'error' | 'workflow-finished'
```

Comment:
I would call it WidgetEventName
And I found these event names in the widget code
```
'error',
'widget-opened',
'widget-closed',
'app-opened',
'app-closed',
'page-changed',
'workflow-finished'
```